src/autocomplete/use-autocomplete.ts

```diff
@@ -79,33 +75,23 @@ export const useAutocomplete = <I>({
 			},
 			[_onChange],
 		),
+		[options, setOptions] = useState<I[]>([]),
```

Comment:
Do we really need a state for options and for loading ? Isn't until enough ?

Reply:
For options we need the state, but I've removed the one for loading.